waspc/src/Wasp/Generator/ServerGenerator/JobGenerator.hs

```diff
@@ -46,9 +48,14 @@ genJobs spec = return $ genJob <$> getJobs spec
   where
     tmplFile = C.asTmplFile $ jobsDirInServerTemplatesDir SP.</> [relfile|_job.js|]
     dstFileFromJobName jobName = jobsDirInServerRootDir SP.</> fromJust (parseRelFile $ jobName ++ ".js")
+
     genJob :: (String, Job) -> FileDraft
     genJob (jobName, job) =
       let (jobPerformFnName, jobPerformFnImportStatement) = getJsImportDetailsForExtFnImport relPosixPathFromJobFileToExtSrcDir $ (J.fn . J.perform) job
+          maybeJobPerformOptions = J.performOptions . J.perform $ job
+          maybeJobSchedule =
```

Comment:
This is also a question for @Martinsos because I've already seen it in several places.

Is it usual (in Haskell) to name values of type `Maybe t` as `maybeFooBar` instead of `fooBar`? Looks a bit like Hungarian notation. If we want to communicate the type, why not use the type system directly (seems more robust):
```haskell
jobPerformOptions:: Maybe JSON = J.performOptions . J.perform $ job
```

Reply:
It is a convention I introduced / made up :D, and I don't do it always, but I do it sometimes when I feel it improves the readability. Often I do it if I am unpacking that value later, since if you have something named `x` and you unpack it from Maybe, you can't name it `x` again. So I name it `maybeX`, which also makes it explicit that this is X with possible failure, and then I have `x` as unpacked value.
Using type -> well that is also fine, doesn't solve the unpacking naming issue though, and it type is long it becomes cumbersome, while this is just `maybe` as prefix.